cumulusci/utils.py

```diff
@@ -382,37 +381,118 @@ def doc_task(task_name, task_config, project_config=None, org_config=None):
     from cumulusci.core.utils import import_global
 
     doc = []
-    doc.append("{}\n==========================================\n".format(task_name))
-    doc.append("**Description:** {}\n".format(task_config.description))
-    doc.append("**Class::** {}\n".format(task_config.class_path))
+    doc.append(f"**{task_name}**\n==========================================\n")
+    doc.append(f"**Description:** {task_config.description}\n")
+    doc.append(f"**Class:** {task_config.class_path}\n")
 
     task_class = import_global(task_config.class_path)
-    if "task_docs" in task_class.__dict__:
-        task_docs = textwrap.dedent(task_class.task_docs.strip("\n"))
-        doc.append(task_docs + "\n")
```

Comment:
@Br4nd0R did you intend to stop including the task_docs? I think it has useful information.

Reply:
These should definitely be in there. I got a little too wrapped up with matching what is in docs for sfdx commands.